.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Type of change
+<!-- Please be sure to add the appropriate label to your PR. -->
```

Comment:
Should we invite folks to add labels if they don't see one that fits?

Reply:
Yes, I just added one 😄